emerlin2caom2/settings_file.py

```diff
@@ -1,4 +1,7 @@
-storage_name = # string path and name of emerlin output, e.g. '/home/h14471mj/e-merlin/casa6_docker/prod/TS8004_C_001_20190801'
-xmldir = # directory where xml file will be generated, e.g. './data/'
-# rootca = # location of rootCA.pem file
-# ska_token = # ska beraer token
+storage_name = '' # string path and name of emerlin output, e.g. '/home/h14471mj/e-merlin/casa6_docker/prod/TS8004_C_001_20190801'
+xmldir = '' # directory where xml file will be generated, e.g. './data/'
+
+upload = True
+replace_old_data = True
+rootca = '' # location of rootCA.pem file
+ska_token = '' # ska beraer token
```

Comment:
Now that this is needed, I don't know what exactly I need to copy/paste or include for the token, so an example would be really helpful.  And will I need to execute the script from the same terminal where bearer token was generated?  Or any other quirks?